components/index/CardsReference/Variant/Card.vue

```diff
@@ -18,6 +18,17 @@
         :last-period="variantData.lastPeriod"
         unit="%"
       >
+        <template #additionalButton>
+          <app-link
+            to="https://www.fukushihoken.metro.tokyo.lg.jp/iryo/kansen/corona_portal/henikabu/screening.html"
+            :class="$style.button"
+          >
+            <span :class="$style.buttonInner">
```

Comment:
```suggestion
            <span :class="$style['button-inner']">
```
実は `nuxt generate` でキャメルケースのclass名が効かないというワナがあって、いまnetlifyのプレビューでスタイルが崩れてしまっています。
上記のような書き方だとうまく通ります。

Reply:
@kaizumaki 
ご指摘ありがとうございます！
70ee00df4abd1ba0b8f9c9d9278e5759975029df にて修正しました！